### DISCUSSION TITLE
Welcome to WolfHUD Discussions!

### DISCUSSION THREAD

**Discussion by jaondayko** [jaondayko/WolfHUD#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by milodakat** [comment]
lmao

**Comment by infologicmation** [comment]
### **[bug report]"steam could not authenticate steam id for the host"**

I use WolfHUD and the other mods such as BLT, NGBTO.

Recenty, almost whenever I join someone's game play(heist), I have encountered the bug.
when I arrive at 'job overview screen', I am immediately kicked by this bug.
and the system says "**steam could not authenticate steam id for the host**".

this bug have happened suddenly since just about a few weeks ago.
and this bug happen with the probabilty more than 70% at the room I tryed to join.

I don't know why this bug happens.
but when I played this game without WolfHUD(=with uninstalling WolfHUD), this bug didn't happen.
so I think WolfHUD cause this bug.

I wish you would fix this bug. please.

-----
*my SuperBLT is latest(1.3.1)
and WolfHUD is latest too(3.3.6b) (downloaded from this site)

**Comment by qwqazpoplm** [comment]
Want to say thank you that you can update wolfhud. It is my favorite hud. It was sad that Kamikaze94 stopped updating it. But now, I am glad that someone takes over it.
